### PR TITLE
Added printHialotoftimes function

### PR DESCRIPTION
**Pull request by jlkunzelman**
@lhernandezcruz

### CODE REVIEW

File: problems.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# TODO: fix this whole function
+
+def printHialotoftimes(n):
```
Comment (lhernandezcruz) [inline problems.rb]: Suggest a better name for the function

File: problems.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	i = 0;
+	j = 0;
+	until i > n  do
+		until j > 3 do
+			puts("Hi");
```
Comment (lhernandezcruz) [inline problems.rb]: Recommend storing “Hi” in a variable to avoid magic values

File: problems.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# TODO: fix this whole function
+
+def printHialotoftimes(n):
+	i = 0;
+	j = 0;
+	until i > n  do
```
Comment (lhernandezcruz) [inline problems.rb]: Suggest combining the two nested loops into one loop for code clarity

File: problems.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# TODO: fix this whole function
```
Comment (lhernandezcruz) [inline problems.rb]: Point out the “TODO” comment that should be removed

File: problems.rb (hunk 2 above)
Comment (jlkunzelman) [reply problems.rb]: yes

**Pull request merged**